lib/puppet/property/ensure.rb

```diff
@@ -54,7 +54,7 @@ def change_to_s(currentvalue, newvalue)
       elsif newvalue == :absent
         return _("removed")
       else
-        return _("%{value0} changed '%{value1}' to '%{value2}'") % { value0: self.name, value1: self.is_to_s(currentvalue), value2: self.should_to_s(newvalue) }
+        return _('%{value0} changed %{value1} to %{value2}') % { value0: self.name, value1: self.is_to_s(currentvalue), value2: self.should_to_s(newvalue) }
```

Comment:
Can we get these "valueX" placeholders replaced with meaningful keys? This was probably my oversight originally, whoops.